ui-pf4/src/main/webapp/src/Constants.tsx

```diff
@@ -74,6 +74,8 @@ export enum AdvancedOptionsFieldKey {
 export class Constants {
   static readonly DEFAULT_PAGE_SIZE = 10;
   static readonly DEFAULT_PAGINATION_OPTIONS = [10, 20, 50, 100];
+  static readonly ALLOWED_APPLICATION_EXTENSIONS =
```

Comment:
I would add also `tar.gz`, `tar.xz` and `7z` as reported in https://issues.redhat.com/browse/WINDUP-2822?focusedCommentId=15430555&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15430555.
Let's wait to find an agreement in Jira before changing anything in the PR

Reply:
@mrizzi right now we can not add ".tar.gz", "tar.xz", nor ".7z" because the CLI doesn't support them. I've already verified it by analyzing applications with those extensions in the Web Console and the CLI, in both cases MTA is not able to recognize the content of those files and decompile them. If we want to add the extensions you suggested we need to add them first to the CLI and then to the Web Console, but we can create another ticket for it.